luxonis_train/nodes/heads/fomo_head.py

```diff
@@ -0,0 +1,106 @@
+import logging
+from typing import Any, List
+
+import torch
+from torch import Tensor, nn
+
+from luxonis_train.enums import TaskType
+from luxonis_train.nodes.base_node import BaseNode
+from luxonis_train.utils import Packet
+
+logger = logging.getLogger(__name__)
+
+
+class FOMOHead(BaseNode[list[Tensor], list[Tensor]]):
+    tasks: list[TaskType] = [TaskType.KEYPOINTS, TaskType.BOUNDINGBOX]
```

Comment:
Similar to the comment above; I'm a bit confused about the output of the model. It predicts a heatmap that's converted to keypoints that somehow define a bounding box of the center of the object? That's how it looks to me in the description image.
If that's the case, we should stick to only one of those tasks (keypoints would probably make the most sense).

Reply:
The issue is that for the metric to function correctly, we need the ground truth bounding boxes to calculate the OKS metric. That is why the dataset must include bounding boxes with a center keypoint for each bounding box.